docs/development.rst

```diff
@@ -37,6 +31,8 @@ During development, once Girder is started via ``python -m girder``, the server
 will reload itself whenever a Python file is modified.
 
 To get the same auto-building behavior for JavaScript, we use ``grunt-watch``.
+Since ``grunt`` is installed in the ``./node_modules/.bin/`` directory, you could
+conveniently update the ``PATH`` running ``export PATH=$(pwd)/node_modules/.bin:$PATH``.
```

Comment:
I strongly agree; some people are fine to install stuff globally for the convenience factor. I suggest: "Alternatively, you could install the grunt command line interface globally so that the `grunt` command it is automatically added to your PATH. If you want to do that, run `npm install -g grunt-cli`. Note that this command requires `sudo` on many systems."